code/modules/antagonists/eldritch_cult/eldritch_structures.dm

```diff
@@ -111,14 +111,14 @@
 	desc = "Collection of unknown symbols, they remind you of days long gone..."
 	icon = 'icons/obj/eldritch.dmi'
 	charges = 1
-	///Owner of the trap
-	var/mob/owner
+	/// Weakref containing the owner of the trap
+	var/datum/weakref/owner
 
 /obj/structure/trap/eldritch/Crossed(atom/movable/AM)
 	if(!isliving(AM))
 		return ..()
 	var/mob/living/living_mob = AM
-	if((owner && living_mob == owner) || IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
+	if(living_mob == owner?.resolve() || IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
```

Comment:
do we want traps to stop working when the knife breaks?

Reply:
the owner is the mob and not the knife, i'm just replicating the old behavior

Reply:
ah, excuse me